src/input_common/sdl/sdl_impl.cpp

```diff
@@ -125,9 +133,16 @@ class SDLJoystick {
         return static_cast<float>(state.axes.at(axis)) / (32767.0f * range);
     }
 
-    bool RumblePlay(u16 amp_low, u16 amp_high) {
+    bool RumblePlay(u16 amp_low, u16 amp_high, u16 freq_low, u16 freq_high) {
         constexpr u32 rumble_max_duration_ms = 1000;
 
+        if (HasHDRumble()) {
+            const u64 effect = (static_cast<u64>(amp_low) << 48) |
```

Comment:
```suggestion
            const u64 effect = (u64{amp_low} << 48) |
```

Ditto for the ones below, since this isn't a narrowing conversion, but a widening one.